src/hooks/useAsset.js

```diff
@@ -5,9 +5,10 @@ import { AssetTypes } from '@rainbow-me/entities';
 
 // To fetch an asset from account assets,
 // generic assets, and uniqueTokens
-export default function useAsset(asset) {
+export default function useAsset(asset, ignorePricing = false) {
```

Comment:
this is done to avoid difficult computation that @terrysahaidak says are the problem in Rainbow